admin_manual/configuration_database/mysql_4byte_support.rst

```diff
@@ -3,7 +3,8 @@ Enabling MySQL 4-byte support
 =============================
 
 .. note::
-
+    Changing of innobd_file_format was depreciated in MySQL Version 8.0 (see dev.mysql.com (Server and Status Variables and Options added, depreciated, or removed in MySqL 8.0).
```

Comment:
In MariaDB 10.2.2 Barracuda is the default and in 10.3 it can't be changed anymore: https://mariadb.com/kb/en/innodb-file-format/

The MySQL deprecation page: https://dev.mysql.com/doc/refman/8.0/en/added-deprecated-removed.html

Reply:
This maybe a bit misleading:

Reply:
Sorry my response was late.  That change should do the trick.

Reply:
Is #5264 fine or does it need further refinements in your opinion?